coralproject/talk
# [CORL-3021]: Update Twitter to X

## What does this PR do?

These updates change mentions of Twitter to X.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Go to the admin. See that under General --> Embedded media, it now says `X post` in the overall configuration description and regarding enable X post embeds. Make sure X post embeds are enabled.

Go to the stream. Paste an X post url. See that you are prompted with `Add this post to the end of your comment?` and the `X` logo. Click `Add post`. See that the post is added to your comment.

Also check that the wording in user preferences around showing/hiding media (including posts) is updated now.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
